src/routes/auth.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+
+const router = express.Router();
+const {
+  signUp,
+  signIn,
+  logOut,
+  getSignUpPage,
+  getSignInPage,
+} = require('../controller/auth');
+
+router.get('/signUp', getSignUpPage);
+router.post('/signUp', signUp);
+router.get('/signIn', getSignInPage);
+router.post('/signIn', signIn);
+router.get('/logOut', logOut);
```

Comment:
And here you can create a separate folder named views or pages for the HTML pages routes